src/doom/g_game.c

```diff
@@ -93,7 +93,7 @@ void	G_DoCompleted (void);
 void	G_DoVictory (void); 
 void	G_DoWorldDone (void); 
 void	G_DoSaveGame (void); 
- 
```

Comment:
Here's one unnecessary white space change.

Reply:
I don't suppose we could all agree to fix the inconsistent whitespace upstream in the doom source tree huh? 😉

Reply:
Sure, I just don't want to add more to this inconsistency.